lib/src/endpoints/playlists.dart

```diff
@@ -36,10 +36,16 @@ class Playlists extends EndpointPaging {
   /// [userId] - the Spotify user ID
   ///
   /// [playlistName] - the name of the new playlist
```

Comment:
Please document the other arguments as well. 😃

Reply:
Done!

Reply:
There are some inconsistencies in this repository when it comes to comments though. For example, the capitalization of the first letter varies, sometimes in [a single doc comment](https://github.com/doodlezucc/spotify-dart/blob/master/lib/src/endpoints/playlists.dart#L118-L121). The added comments should fit in.

Reply:
Yeah, that's true - but let's pick it up as we go. Thanks for your engagement!! 😄